fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/logging/LogMessageKeys.java

```diff
@@ -32,14 +32,28 @@
 public enum LogMessageKeys {
     // general keys
     TITLE("ttl"),
+    CLUSTER("cluster"),
     SUBSPACE("subspace"),
     SUBSPACE_KEY("subspace_key"),
+    INDEX_SUBSPACE("indexSubspace"),
```

Comment:
We weren't very consistent in our code, using camel-case in some places and underscores in another.  I think for consistency, we should use this as an opportunity to be consistent, so this should be `index_subspace` and, similarly the other camel-case's here should be updated.

Reply:
I suppose the concern would be if someone were using the log names as part of their monitoring routine. (Like, if they had scripts or something that searched through logs for key/value pairs with certain names.)

I think it's still worth it to normalize on underscores, but that's the risk.

Reply:
Checked with concerned parties and found no known dependency on existing log keys from fdb-record-layer. After consultation, taking the risk.